### PR TITLE
Update slackbot-guysgals-responses.md

### PR DESCRIPTION
**Pull request by zkrige**
added trigger phrases

### CODE REVIEW

File: slackbot-guysgals-responses.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # When someone says
 
-you guys, these guys, my guys, those guys, hey guys, hi guys, the guys, these guys, thanks guys, guyz
+you guys, these guys, my guys, those guys, hey guys, hi guys, the guys, these guys, thanks guys, guyz, your guys, your guyz
```
Comment (calvin-brizzi) [inline slackbot-guysgals-responses.md]: guyz gets caught always, no need to have variants :) Same with galz below.

**Comment by zkrige** [comment]
updated for "your guys" and "your gals"

as in "your guys response is fantastic"

**Comment by n1c** [comment]
Why's this removing `guyz`, `galz`?

**Comment by JohnRoux** [comment]
Yeah, adding `your [guys/gals]` is cool

Not sure the reasoning for removing `guyz/galz`

**Comment by zkrige** [comment]
🤦‍♂️

**Pull request closed without merging**